Sources/Plasma/NucleusLib/pnAsyncCoreExe/CMakeLists.txt

```diff
@@ -28,12 +23,16 @@ set(pnAsyncCoreExe_PRIVATE_WIN32
     Private/Win32/pnAceW32Thread.cpp
 )
 
-# End questionable part
-
 add_library(pnAsyncCoreExe STATIC
             ${pnAsyncCoreExe_SOURCES} ${pnAsyncCoreExe_HEADERS}
-            ${pnAsyncCoreExe_PRIVATE} ${pnAsyncCoreExe_PRIVATE_NT}
-            ${pnAsyncCoreExe_PRIVATE_WIN32})
+            ${pnAsyncCoreExe_PRIVATE})
+
+if(WIN32)
+    target_sources(pnAsyncCoreExe PRIVATE
+        ${pnAsyncCoreExe_PRIVATE_NT}
+        ${pnAsyncCoreExe_PRIVATE_WIN32}
+    )
+endif()
```

Comment:
This could also be a generator expression in the main `add_library()` call:
```cmake
add_library(foobar STATIC $<$<PLATFORM_ID:Windows>:${WINDOWS_SOURCES}>)
```

Reply:
Eh, it could...  But I don't really see the benefit of a generator expression in this case, especially since this whole block will eventually go away (including the Win32Thread bit).